app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,13 @@
+class GuestsController < ApplicationController
+
+  def show
+    @guest = Guest.find(params[:id])
+  end
+
+  def update
+    guest = Guest.find(params[:id])
```

Comment:
This implementation looks solid, but the most RESTful location for this logic would actually be in a ReservationsController. The logic we would expect in a "guests#update" action would be updates to any attributes on the actual guest resource. In this case, you're creating a joins record between a guest and a room, so you're upating the guest resource itself or its attributes. So, you could create a joins controller and call it ReservationsController and then put this logic in a `create` action, since what you're doing is `creating` a joins record!